airbyte-cdk/python/airbyte_cdk/entrypoint.py

```diff
@@ -52,6 +54,7 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         # set up parent parsers
         parent_parser = argparse.ArgumentParser(add_help=False)
         parent_parser.add_argument("--debug", action="store_true", help="enables detailed debug logs related to the sync")
+        parent_parser.add_argument("--flush", type=float, default=0, help="flush interval in seconds (0 to disable)")
```

Comment:
I think it is ok to set this parameter using the `ENV` with `default=0.1`

Reply:
Shouldn't backwar compatibilty be the king? It's probably fine but why do something potentially destructive if it can be avoided? Gonna look into using ENV as a variant of setting that, great point.

Reply:
> Shouldn't backward compatibilty be the king?

generally yes.

> why do something potentially destructive if it can be avoided?

I dont see any _destructive_ cases here, especially if we'll catch all stdout.


```suggestion
        flush_interval = os.getenv("flush_interval", 0.1)
```